### PR TITLE
First Question

### PR DESCRIPTION
**Pull request by CooperW12**

### CODE REVIEW

**Review by Team1710CTO — DISMISSED** [review]

**Review by TheChar — DISMISSED** [review]

**Review by rahidas04 — DISMISSED** [review]

**Review by Team1710CTO — DISMISSED** [review]

File: MyClass.java [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
-    }
+public class MyClass {
+	public static void main(String args[]) {
+	   
+	}
+	   public static int pot(){
+	     Scanner potScanner = new Scanner(System.in);
+	     System.out.println("How many people are on the team?");
+	     //"pot" means people on team
+	     int pot = potScanner.nextInt();
```
Comment (smoore190) [inline MyClass.java]: When dealing with user inputs think about what the limits are and if you need to do any error handling.  For example does nextInt() allow negative numbers?

**Review by rahidas04 — DISMISSED** [review]
I like the comments you made! It makes the code easier to understand!

**Review by Team1710CTO — APPROVED** [review]

**Comment by CooperW12** [comment]
I changed the code to account for negative numbers as a response

**Review by rahidas04 — APPROVED** [review]

File: MyClass.java [hunk 2]
```diff
@@ -17,6 +17,9 @@ public static int pot(){
 	    Scanner potScanner = new Scanner(System.in); //takes user input
 	    System.out.println("How many people are on the team in total?"); //prints question
 	    int pot = potScanner.nextInt(); //"pot" means people on team
+            if (pot < 0){
```
Comment (smoore190) [inline MyClass.java:20]: Nice!

**Review by Andrew-K961 — APPROVED** [review]
I would add user feedback after the user inputs a number. Other than that, seems to work well.

**Pull request merged**